### PR TITLE
1077 - node exporter deployment

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: Monitoring

**Summary**: Deploy node-exporter

**Acceptance criteria**: node-exporter pod is up and running

### CODE REVIEW

File: salt/metalk8s/addons/monitoring/grafana/deployed.sls [hunk 1]
```diff
@@ -7476,6 +7489,13 @@ spec:
       - configMap:
           name: grafana-dashboard-statefulset
         name: grafana-dashboard-statefulset
+      tolerations:
```
Comment (NicolasT) [inline salt/metalk8s/addons/monitoring/grafana/deployed.sls:7492]: Is it possible to also specify a `nodeSelector` for `node-role.kubernetes.io/infra: ''`? This also applies for all other 'infra' services you're deploying here.

Comment (slaperche-scality) [reply salt/metalk8s/addons/monitoring/grafana/deployed.sls:7492]: Tracked in #1169.

**Comment by slaperche-scality** [comment]
@bert-e reset

**Comment by slaperche-scality** [comment]
@bert-e reset

**Review by slaperche-scality — APPROVED** [review]

**Comment by slaperche-scality** [comment]
@tcarmet is it possible to bypass @alexandre-allard-scality self-approval on this one?

We need it for MVP and he is on holidays.

**Comment by alexandre-allard** [comment]
@bert-e approve

**Comment by slaperche-scality** [comment]
thanks @alexandre-allard-scality :slightly_smiling_face:

**Pull request merged**